openedx/adg/lms/webinars/helpers.py

```diff
@@ -234,37 +235,47 @@ def remove_emails_duplicate_in_other_list(email_list, reference_email_list):
     return [email for email in email_list if email not in reference_email_list]
 
 
-def update_webinar_team_registrations(webinar_form):
+def get_newly_added_and_removed_team_members(webinar_form):
     """
-    Updates webinar team registrations if webinar presenter, cohosts or panelists are updated.
+    Gets a list of newly added and removed co-hosts, panelists and presenter
 
     Args:
         webinar_form (Form): Model form with updated data.
 
     Returns:
-        None
+        new_members (list): list of newly added co-hosts, panelists and presenter
+        removed_members (list): list of removed co-hosts, panelists and presenter
     """
-    from openedx.adg.lms.webinars.models import WebinarRegistration
-
     webinar = webinar_form.instance
     cleaned_data = webinar_form.cleaned_data
     old_team = webinar.webinar_team()
     new_team = set(chain(cleaned_data['co_hosts'], cleaned_data['panelists'], {cleaned_data['presenter']}))
 
-    newly_added_members = list(new_team - old_team)
+    new_members = list(new_team - old_team)
```

Comment:
What if the old team and the new team has an overlapping member? Would that member still be receiving their reminder emails and/or update emails?

Reply:
if old team and new team has an overlapping member, it would mean that that member is neither removed nor added. Yes, it's behaviour won't be impacted. In this function, we are only extracted new and removed members.

Reply:
That's great! Thanks for the clarification!